llvm/test/CodeGen/AMDGPU/mfma-loop.ll

```diff
@@ -4,7 +4,8 @@
 
 ; GCN-LABEL: {{^}}test_mfma_loop_zeroinit:
 
-; GCN-COUNT-32: v_accvgpr_write_b32 a{{[0-9]+}}, 0{{$}}
+; GCN:          v_mov_b32_e32 v0, 0{{$}}
+; GCN-COUNT-32: v_accvgpr_write_b32 a{{[0-9]+}}, v0{{$}}
```

Comment:
Lots of agpr test cases show regressions like this where we fail to fold inline constants into v_accvgpr_write_b32.